beaker-vue/src/lib/UICell.vue

```diff
@@ -147,30 +170,4 @@ const executeAndMove = () => {
     opacity: 0.2;
 }
 
-.drag-source {
```

Comment:
I'm undecided on this. Had thought that the parent, which attaches the classes now, should define the style rules. Additionally, BeakerCell could be single-cell scenario, in which case these don't matter. But now I realized that BeakerCell is still aware of its dragging capability (props.dragEnabled, draggable handle) so maybe I should revert this.